Cargo.toml

```diff
@@ -354,6 +354,7 @@ serde = { version = "1", features = ["derive"] }
 bytemuck = "1.7"
 # Needed to poll Task examples
 futures-lite = "2.0.1"
+async-std = "*"
```

Comment:
Is this our effective recommendation for this kind of stuff?

Reply:
nop I 🤦  there